### PR TITLE
Updated packages versions

### PR DESCRIPTION
**Pull request by KyryloVolotovskyi**

### CODE REVIEW

File: packages/performance-tests/package.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
     "@types/node": "14.14.31",
     "@types/yargs": "^12.0.5",
     "chai": "^4.3.6",
+    "encoding": "^0.1.13",
```
Comment (MichaelBelousov) [inline packages/performance-tests/package.json:44]: don't introduce dependencies we don't use... if you're making a monorepo on top of this repo, alter its hoisting settings to access this.

File: packages/performance-tests/package.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
     "@itwin/core-quantity": "3.6.0 - 4.0.999",
     "@itwin/imodel-transformer": "workspace:*",
     "@itwin/ecschema-metadata": "3.6.0 - 4.0.999",
-    "@itwin/imodels-access-backend": "^2.2.1",
+    "@itwin/imodels-access-backend": "2.3.0",
```
Comment (MichaelBelousov) [inline packages/performance-tests/package.json:24]: why pin this? Why not move it up to `"^2.3.0"`?

File: packages/performance-tests/test/transformers/RawForkOperations.ts [hunk 3]
```diff
@@ -1,8 +1,9 @@
-import { initializeBranchProvenance } from "@itwin/imodel-transformer";
+import { IModelDb } from "@itwin/core-backend";
 import { TestTransformerModule, TransformRunner } from "../TestTransformerModule";
+import { initializeBranchProvenance } from "@itwin/imodel-transformer";
 
 const rawForkOperationsTestModule: TestTransformerModule = {
-  async createForkInitTransform(sourceDb, targetDb): Promise<TransformRunner> {
+  async createForkInitTransform(sourceDb: IModelDb, targetDb: IModelDb): Promise<TransformRunner> {
```
Comment (MichaelBelousov) [inline packages/performance-tests/test/transformers/RawForkOperations.ts:6]: these file changes don't seem necessary?

File: packages/performance-tests/test/transformers/RawForkCreateFedGuids.ts [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
+import { initializeBranchProvenance } from "@itwin/imodel-transformer";
 import { setToStandalone } from "../iModelUtils";
 import path from "path";
-import { StandaloneDb } from "@itwin/core-backend";
 
 const outputDir = path.join(__dirname, ".output");
 
 const rawForkCreateFedGuidsTestModule: TestTransformerModule = {
-  async createForkInitTransform(sourceDb, targetDb): Promise<TransformRunner> {
+  async createForkInitTransform(sourceDb: IModelDb, targetDb: IModelDb): Promise<TransformRunner> {
```
Comment (MichaelBelousov) [inline packages/performance-tests/test/transformers/RawForkCreateFedGuids.ts:12]: why are these changes necessary?

Comment (KyryloVolotovskyi) [reply packages/performance-tests/test/transformers/RawForkCreateFedGuids.ts:12]: `NativeTransformer` has those specified, so I thought it would be better to make it clear what type are source and target dbs.
